nvme.c

```diff
@@ -9523,10 +9523,22 @@ static int tls_key(int argc, char **argv, struct command *command, struct plugin
 		nvme_show_error("Must specify either --import, --export or --revoke");
 		return -EINVAL;
 	} else if (cfg.export) {
+		if (!cfg.keyfile) {
```

Comment:
This will prevent to print it to stdout, which is very helpful when debugging. Also `nvme tls --export > file` doesn't work anymore.

Reply:
Ah ok...I didn't realize those were applicable. I will go ahead and remove this check now. Also, I think it makes sense now to consolidate these minor patches to a single one patch itself.

Reply:
Thanks a lot!